README.md

```diff
@@ -41,7 +35,116 @@ Specifically, some of the major features that are working are as follows:
 - Monster spawns
 - Quests
 
-Requirements
+Installation with Docker
```

Comment:
I'd prefer you revert the changes to the readme, which is supposed to be both concise and general purpose. Long explanations about the setup are better suited for the docs folder.